python/py-async-lru/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-async-lru
+version             2.0.4
+revision            0
+categories-append   devel
+license             BSD
+supported_archs     noarch
+platforms           {darwin any}
+
+python.versions     38 39 310 311
+
+maintainers         nomaintainer
+
+description         A port of Python’s built-in functools.lru_cache function for asyncio
+long_description    {*}${description}
+
+homepage            https://github.com/aio-libs/async-lru
+
+checksums           rmd160  597fb64512f28317f2fcf5b3ca5ac424d48591d4 \
+                    sha256  b8a59a5df60805ff63220b2a0c5b5393da5521b113cd5465a44eb037d81a5627 \
+                    size    10019
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools
+
+    if {${python.version} < 311} {
+        depends_lib-append    port:py${python.version}-typing_extensions
```

Comment:
this is not aligned correctly with four spaces as stated in the modeline. It should be aligned with the `port:` statement above (starting at column 29).